doc/api/modules.md

```diff
@@ -368,7 +368,7 @@ example, then `require('./some-library')` would attempt to load:
 
 <!--type=misc-->
 
-If the module identifier passed to `require()` is not a native module,
+If the module identifier passed to `require()` is not a core module,
```

Comment:
Thanks @richardlau. I'm OK then with the text proposed, but suggest the word "core module" be linked to the definition.